miden-lib/asm/miden/tx.masm

```diff
@@ -90,3 +90,24 @@ export.add_asset_to_note
     movdn.4 dropw
     # => [note_ptr]
 end
+
+#! Returns the RECIPIENT for a specified SERIAL_NUM, SCRIPT_HASH, and input
+#!
+#! Inputs: [SERIAL_NUM, SCRIPT_HASH, input]
+#! Outputs: [RECIPIENT]
+#!
+#! Only allows a single input currently   
```

Comment:
I think for now, this is ok. I expect this to be used in the context of a P2ID script which requires only one input anyway.